modules/nw-gcp-installing-global-access-configuration.adoc

```diff
@@ -5,7 +5,7 @@
 
 [id="nw-gcp-global-access-configuration_{context}"]
 = Create an Ingress Controller with global access on GCP
-You can create an Ingress Controller with global access on a new GCP cluster when your. Global access is only available to Ingress Controllers using internal load balancers.
+You can create an Ingress Controller that has with global access to a Google Cloud Platform (GCP) cluster. Global access is only available to Ingress Controllers using internal load balancers.
```

Comment:
@EricPonvelle should that be _that has with global_?

Reply:
Changed it to "that has global access." Thanks for the catch!